client/constants/messages.js

```diff
@@ -45,12 +45,24 @@ const messages = {
   FETCH_SYNC_RECORDS: _, /* @param Array.<string> categoryNames, @param {number} startAt (in seconds) */
   /**
    * webview -> browser
-   * browser must update its local values with the new sync records, performing
-   * conflict-resolution as necessary.
+   * browser should find objects with matching objectIds then call
+   * RESOLVE_SYNC_RECORDS to perform conflict resolution.
    */
   RECEIVE_SYNC_RECORDS: _, /* @param {string} categoryName, @param {Array.<Object>} records */
```

Comment:
sync's requestUtil still needs to make separate s3 ListObject requests per category, so in case a user syncs a subset of the options it would be a little faster.

we could not worry about that, but if it's already implemented i think it's ok to leave in categories for Receive?